tchannel-core/src/main/java/com/uber/tchannel/tracing/Tracing.java

```diff
@@ -31,6 +32,8 @@
 import io.opentracing.Tracer;
 import io.opentracing.propagation.Format;
 import io.opentracing.tag.Tags;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
```

Comment:
is this a sensible dependency?

Reply:
We use these in other places, too. The benefit is the clear contract for the arguments / return values.